compiler/modules/print.llvm/common.bal

```diff
@@ -39,11 +39,12 @@ public function arrayType(Type ty, int elementCount) returns ArrayType {
 
 // Corresponds to llvm::StructType
 public type StructType readonly & record {
+    string? name;
```

Comment:
Why is `name` part of the struct type?

Reply:
I thought `name` to be a part of the type similar to it's element types.

Reply:
I don't think it belongs: you are mapping from a name to a struct type.